tests/PHPUnit/Integration/ArchiveProcessor/LoaderTest.php

```diff
@@ -181,28 +177,12 @@ public function test_pluginOnlyArchivingDoesNotRelaunchChildArchives_whenReusing
             ],
             [
                 'idarchive' => '3',
```

Comment:
The changes to this files are caused by the fact, that the dependent archives are no longer been processed when no goals are available and ecommerce is disabled. Therefor the `Goals` archives (for segments) disappeared. The `VisitsSummary` archives are still present, as they are archived by the `VisitFrequency` archiver.